src/v2/Components/Toast/ToastComponent.tsx

```diff
@@ -0,0 +1,88 @@
+import styled from "styled-components"
+import colors from "../../Assets/Colors"
+import { color, Flex, Text, themeProps } from "@artsy/palette"
+import React, { useEffect, useState } from "react"
+import { useNavBarHeight } from "../NavBar/useNavBarHeight"
+import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+
+const ToastContainer = styled.div<{
```

Comment:
We should avoid CSS and `divs` (etc) where possible and instead leverage Palette, which has a lot of these properties:
```
const ToastContainer = styled(Box)`
  ...
```

Reply:
I can't use `styled(Box)` because, if I'm not mistaken `styled` receives only native html elements. anyway I got rid of `styled-component` by using Box with props

Reply:
Cool 👍 

`styled(SomeComponent)` works with any styled component. Its like `class Foo extends Bar`

Reply:
(You can use `styled` with anything that passes the `className` into the component)